bitters.gemspec

```
@@ -3,11 +3,11 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require "bitters/version"
 
 Gem::Specification.new do |s|
-  s.add_development_dependency "bundler", "~> 1.3"
+  s.add_development_dependency "bundler", "~> 2.1"
```

Unless we require specific features of Bunder `2.1` that `2.0` doesn’t have, thoughts on loosening this a bit?

```suggestion
  s.add_development_dependency "bundler", "~> 2.0"
```

---

Resolved by e79c817cf5ac748b641a198520936c15af8c1e12.